devices/datek.js

```diff
@@ -34,46 +34,52 @@ module.exports = [
         zigbeeModel: ['Meter Reader'],
         model: 'HSE2905E',
         vendor: 'Datek',
-        description: 'Datek Eva AMS HAN power-meter sensor',
+        description: 'Datek Eva AMS HAN power-meter sensor Re-Worked',
```

Comment:
I don't think adding re-worked here adds any value. Better to write something for the changelog.